presto-snowflake/src/main/java/io/prestosql/plugin/snowflake/SnowflakeConfig.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.snowflake;
+
+import io.airlift.configuration.Config;
+
+import java.util.Optional;
+
+public class SnowflakeConfig
+{
+    private Optional<String> account;
+    private Optional<String> database;
+    private Optional<String> role = Optional.of("PUBLIC");
+    private Optional<String> warehouse;
+
+    public Optional<String> getAccount()
+    {
+        return account;
+    }
+
+    @Config("snowflake.account")
+    public SnowflakeConfig setAccount(String account)
+    {
+        this.account = Optional.ofNullable(account);
```

Comment:
```suggestion
        this.account = Optional.of(account);
```

Reply:
I think we should keep these as-is if you agree, because they are in fact nullable. It's possible for the user to specify these options in the connection string, or through default properties in Snowflake. With that in mind I also removed the default role that was set to "PUBLIC" for these reasons. It makes more sense not to force a default on the user.